contracts/upgradeable_contracts/OwnedUpgradeability.sol

```diff
@@ -6,7 +6,7 @@ import "../IOwnedUpgradeabilityProxy.sol";
 contract OwnedUpgradeability {
 
     function upgradeabilityAdmin() public view returns (address) {
```

Comment:
Do we need `upgradeabilityAdmin` in this case at all? What impact for other components could be if we leave `upgradeabilityOwner` only (assuming that it is public)?

Reply:
`upgradeabilityAdmin` is only used on `onlyIfOwnerOfProxy` on the contracts and no other bridge components uses it. If it is needed, other components can call `upgradeabilityOwner` directly.
In this case, I think we can remove `upgradeabilityAdmin` and redefine `onlyIfOwnerOfProxy` as:
```solidity
    modifier onlyIfOwnerOfProxy() {
        require(msg.sender == IOwnedUpgradeabilityProxy(this).upgradeabilityOwner());
        _;
    }
```

Reply:
Applied changes https://github.com/poanetwork/poa-bridge-contracts/pull/198/commits/41f9ba8cf67e04817a7d59f7a8a865c83f30d1a8